dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/SearchServiceBean.java

```diff
@@ -676,7 +676,7 @@ public SolrQueryResponse search(DataverseRequest dataverseRequest, List<Datavers
                         stringBuilder.append(getCapitalizedName(part.toLowerCase()) + " ");
                     }
                     String friendlyNameWithTrailingSpace = stringBuilder.toString();
-                    String friendlyName = getLocaleFacetName(friendlyNameWithTrailingSpace.replaceAll(" $", ""));
+                    String friendlyName = getLocaleFacetCategoryName(facetField.getName());
```

Comment:
friendlyNameWithTrailingSpace var can be removed and after that stringBuilder and parts vars